index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
Nice review comment. If you look closely, you will see the structure of this comment:
- It contains the main issue which is extra space
- Then it contains, **_why_** the there should be no extra space.

You did well here.